app/frontend/pairings/Pairing.svelte

```diff
@@ -5,6 +5,12 @@
 
     export let round: Round;
     export let pairing: Pairing;
+    export let left_player = pairing.player1
+    export let right_player = pairing.player2
+    if (pairing.is_single_sided_swiss && pairing.player2.side_label == '(Corp)') {
```

Comment:
If we want logic based on the player's side for the pairing, maybe we should use the side enum rather than a label? We could generate the label on the client side if that's neater?

Reply:
I'm going to leave that to a future refactoring, if we want to do it.